test/validation/api/ipsec/ipsec.c

```diff
@@ -370,6 +458,48 @@ void ipsec_sa_param_fill(odp_ipsec_sa_param_t *param,
 	param->lifetime.hard_limit.packets = 10000 * 1000;
 }
 
+static void ipsec_status_event_handle(odp_event_t ev_status,
+				      odp_ipsec_sa_t sa,
+				      enum ipsec_test_sa_expiry sa_expiry)
+{
+	odp_ipsec_status_t status = {
+		.id = 0,
+		.sa = ODP_IPSEC_SA_INVALID,
+		.result = 0,
+		.warn.all = 0,
+	};
+
+	CU_ASSERT_FATAL(ODP_EVENT_INVALID != ev_status);
+	CU_ASSERT_EQUAL(1, odp_event_is_valid(ev_status));
+	CU_ASSERT_EQUAL(ODP_EVENT_IPSEC_STATUS, odp_event_type(ev_status));
```

Comment:
This should probably be a fatal assert since odp_ipsec_status() must not be called for wrong types of events.

Reply:
Ack. Made it CU_ASSERT_EQUAL_FATAL().